views/prisonerProfile/prisonerWorkAndSkills/prisonerWorkInsidePrisonDetails.njk

```diff
@@ -0,0 +1,96 @@
+{% extends "../../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "../../macros/printLink.njk" import printLink %}
+
+
+{% set title =  "Work and activities inside prison for the last 12 months" %}
+{% set workDetailsList =  workHistoryInsidePrison.content %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: "/"
+      },
+      {
+        text: breadcrumbPrisonerName,
+        href: profileUrl
+      }
+    ],
+    classes: "govuk-!-display-none-print"
+  }) }}
+{% endblock %}
+
+{% set rows = [] %}
+{% for job in workDetailsList %}
+  {% set rows = (rows.push([
+    { text: job.role },
+    { text: job.location },
+    { text: job.startDate | formatTimestampToDate, attributes: { "data-sort-value": job.startDate | formatDate("X") } },
```

Comment:
What does X mean? Is this required is the startDate is always set?

Reply:
It's the way I've seen it's done elsewhere in the repo - it's a unix format - thought I'd stay consistent with existing code. The date sorting doesn't seem to work otherwise